packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-offer.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Component,
+  h,
+  Prop,
+  State,
+  Host,
+  Element,
+  Fragment,
+} from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import { HTMLStencilElement } from '@stencil/core/internal';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import {
+  ServiceProperties,
+  ServiceInfos,
+  Translations,
+  ServiceDetails,
+} from './msc-billing.types';
+import { getChangeOfferUrl } from './urls';
+
+export interface IMscBillingOffer {
+  serviceType: string;
+  servicePath: string;
+  serviceInfos?: ServiceInfos;
+  serviceDetails?: ServiceDetails;
+  changeOfferDataTracking?: string;
+  localeStrings: Translations;
+}
+
+@Component({
+  tag: 'msc-billing-offer',
+  styleUrl: 'msc-billing-offer.scss',
+  shadow: true,
+})
+export class MscBillingOffer implements IMscBillingOffer {
+  @Element() host!: HTMLStencilElement;
+
+  @Prop() serviceType: string;
+
+  @Prop() servicePath: string;
+
+  @Prop() serviceInfos: ServiceInfos;
+
+  @Prop() serviceDetails?: ServiceDetails;
+
+  @Prop() changeOfferDataTracking?: string;
+
+  @Prop() localeStrings: Translations;
+
+  @State() serviceProperties: ServiceProperties;
+
+  async componentWillLoad() {
+    this.serviceProperties = await apiClient.v6
+      .get(this.servicePath)
+      .then((response) => response.data);
+  }
+
+  private isUpgradableOffer() {
+    return (
+      this.serviceType !== 'DOMAIN' && !this.serviceDetails?.parentServiceId
+    );
+  }
```

Comment:
**nitpick:** Super-component cannot maintain the list of products like this. It can accept an input whether the owner can be changed or not.

Reply:
https://github.com/ovh/manager/pull/9692/files/5f903977980beb4137e1f9843da79d9fff8fe8d8#r1328340236

Here it is about offers
And why would it be better to put the domain logic inside each apps instead of keeping it in the component ?